tests/restapi/test_restapi.py

```diff
@@ -341,6 +341,8 @@ def test_data_path(construct_url, vlan_members, cleanup_after_testing):
     pytest_assert(r.status_code == 204)
 
     # Verify routes
+    # Add some delay before query
```

Comment:
why not retry instead?

Reply:
Suggest using wait_until wrapper.

Reply:
This sleep logic is implemented 5 times during the test, I added it as it was missing in this part.
As for refactoring the test to use wait_until instead of static sleeps- I agree with the approach but this can be handled on a different PR and do it broadly.